common/src/main/scala/org/apache/comet/CometConf.scala

```diff
@@ -625,36 +628,3 @@ private[comet] case class ConfigBuilder(key: String) {
 private object ConfigEntry {
   val UNDEFINED = "<undefined>"
 }
-
-/**
- * Utility for generating markdown documentation from the configs.
- *
- * This is invoked when running `mvn clean package -DskipTests`.
- */
```

Comment:
This code had to move to the spark module so that it can access the cast information